get-shit-done/workflows/mvp-phase.md

```
@@ -0,0 +1,208 @@
+PHASE_FOUND=$(echo "$PHASE_INFO" | jq -r '.found')
+PHASE_NAME=$(echo "$PHASE_INFO" | jq -r '.phase_name')
+PHASE_GOAL=$(echo "$PHASE_INFO" | jq -r '.goal')
+PHASE_MODE=$(echo "$PHASE_INFO" | jq -r '.mode // ""')
+PHASE_COMPLETE=$(echo "$PHASE_INFO" | jq -r '.roadmap_complete // false')
+```
+
+If `PHASE_FOUND` is `false`: error and exit. Suggest `/gsd add-phase` or `/gsd insert-phase` to create the phase first.
+
+**Status guard.** If the phase is `in_progress` (has plans but not complete) or `completed`, refuse unless `--force` is in `$ARGUMENTS`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)